plugins/auth-backend/src/identity/TokenFactory.ts

```diff
@@ -23,6 +23,7 @@ import { LoggerService } from '@backstage/backend-plugin-api';
 import { AnyJWK, KeyStore, TokenIssuer, TokenParams } from './types';
 
 const MS_IN_S = 1000;
+const MAX_TOKEN_LENGTH = 32768; // At 64 bytes per entity ref this still leaves room for about 500 entities
```

Comment:
Bumped up to 32k, since people have been hitting 4k in production setups, so 8k seemed a bit low. The intention is for this to never be hit in production setups without bugs